packages/cfd/src/Containers/cfd-password-manager-modal.tsx

```diff
@@ -138,15 +152,15 @@ const CFDPasswordManagerTabContent = ({
     platform,
     onChangeActiveTabIndex,
     account_group,
-}) => {
-    const [active_tab_index, setActiveTabIndex] = React.useState(0);
-    const [error_message_investor, setErrorMessageInvestor] = React.useState('');
-    const [is_submit_success_investor, setSubmitSuccessInvestor] = React.useState(false);
+}: TCFDPasswordManagerTabContent) => {
```

Comment:
since we do not use `password_type` prop in the `CFDPasswordManagerTabContent` function, we can remove it from `TCFDPasswordManagerTabContent` types